mcux/mcux-sdk/tools/cmake_toolchain_files/armgcc_aarch64.cmake

```diff
@@ -48,9 +48,36 @@ SET(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -O3 " CACHE INTERNAL "cx
 SET(CMAKE_ASM_FLAGS_RELEASE "${CMAKE_ASM_FLAGS_RELEASE}" CACHE INTERNAL "asm compiler flags release")
```

Comment:
Care to explain more in the commit message why this commit is needed. What is the actual update? I could see that you add floating point option but it is not clear from the commit message.